src/lib/remix/components/Network.svelte

```diff
@@ -1,38 +1,48 @@
 <script lang="ts">
   import {
-    chainDisplayNames,
+    getNetworkDisplayName,
     isProductionNetwork,
+    type NetworkResponse,
     type TenantNetworkResponse,
   } from "$lib/models/network";
   import type { DropdownItem } from "$lib/models/ui";
   import { globalState } from "$lib/state/state.svelte";
   import Dropdown from "./shared/Dropdown.svelte";
 
   type Props = {
-    onSelected: (network: string) => void;
+    onSelected: (network: NetworkResponse | TenantNetworkResponse) => void;
   };
   const { onSelected }: Props = $props();
 
-  const getNetworkGroup = (network: string | TenantNetworkResponse) => {
-    const type = typeof network !== "string" ? network.networkType : undefined;
+  const getNetworkGroup = (network: NetworkResponse | TenantNetworkResponse) => {
+    const type = network.networkType
+
     if (type === 'fork') return 'Forked Networks';
     if (type === 'private') return 'Private Networks';
 
     const isProduction = isProductionNetwork(network);
     return isProduction ? 'Production Networks' : 'Test Networks';
   };
 
-  const networkToDropdownItem = (network: string | TenantNetworkResponse) => {
-    const n = typeof network === "string" ? network : network.name;
+  const networkToDropdownItem = (network: NetworkResponse | TenantNetworkResponse) => {
+ 
     return { 
-      label: chainDisplayNames[n] ?? n, 
+      label: getNetworkDisplayName(network), 
       value: network, 
       group: getNetworkGroup(network),
     };
   };
 
   // network selection logic
-  let network = $state("");
+  let network = $state<NetworkResponse | TenantNetworkResponse>({
+    name: "mainnet",
+    displayName: "Mainnet",
```

Comment:
Does this set the default selection?  I don't think it should have any default, since we can't assume what network the user wants to use.

If this is needed, we should name this either "Ethereum Mainnet" (previous name in the plugin) or just "Ethereum" (to match the name in Defender).
```suggestion
    displayName: "Ethereum",
```